diff -ru php-7.0.11-orig/acinclude.m4 php-7.0.11/acinclude.m4
--- php-7.0.11-orig/acinclude.m4	2016-09-13 13:52:22.000000000 -0500
+++ php-7.0.11/acinclude.m4	2016-10-10 09:09:40.923655467 -0500
@@ -2685,9 +2685,9 @@
 
   echo "'[$]0' \\" >> $1
   if test `expr -- [$]0 : "'.*"` = 0; then
-    CONFIGURE_COMMAND="$CONFIGURE_COMMAND '[$]0'"
+    CONFIGURE_COMMAND="$CONFIGURE_COMMAND '[$]0' [$]*"
   else 
-    CONFIGURE_COMMAND="$CONFIGURE_COMMAND [$]0"
+    CONFIGURE_COMMAND="$CONFIGURE_COMMAND [$]0 [$]*"
   fi
 
   for arg in $clean_configure_args; do
diff -ru php-7.0.11-orig/aclocal.m4 php-7.0.11/aclocal.m4
--- php-7.0.11-orig/aclocal.m4	2016-09-13 13:52:20.000000000 -0500
+++ php-7.0.11/aclocal.m4	2016-10-10 09:09:33.309670250 -0500
@@ -2685,9 +2685,9 @@
 
   echo "'[$]0' \\" >> $1
   if test `expr -- [$]0 : "'.*"` = 0; then
-    CONFIGURE_COMMAND="$CONFIGURE_COMMAND '[$]0'"
+    CONFIGURE_COMMAND="$CONFIGURE_COMMAND '[$]0' [$]*"
   else 
-    CONFIGURE_COMMAND="$CONFIGURE_COMMAND [$]0"
+    CONFIGURE_COMMAND="$CONFIGURE_COMMAND [$]0 [$]*"
   fi
 
   for arg in $clean_configure_args; do
diff -ru php-7.0.11-orig/configure php-7.0.11/configure
--- php-7.0.11-orig/configure	2016-09-13 13:52:26.000000000 -0500
+++ php-7.0.11/configure	2016-10-10 09:09:47.991641743 -0500
@@ -3455,9 +3455,9 @@
 
   echo "'$0' \\" >> config.nice
   if test `expr -- $0 : "'.*"` = 0; then
-    CONFIGURE_COMMAND="$CONFIGURE_COMMAND '$0'"
+    CONFIGURE_COMMAND="$CONFIGURE_COMMAND '$0' [$]*"
   else
-    CONFIGURE_COMMAND="$CONFIGURE_COMMAND $0"
+    CONFIGURE_COMMAND="$CONFIGURE_COMMAND $0 [$]*"
   fi
 
   for arg in $clean_configure_args; do
